app/code/Ludmila/LSNewClasses/Model/ShowConstants.phtml

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
If it is a model, it should have the name ShowConstants.php, but not ShowConstants.phtml, it's not a template